src/airport.js

```diff
@@ -1,26 +1,19 @@
 const Plane = require(`./plane`);
+const Weather = require('./weather')
 
 class Airport {
-  // here's a starting point for you
-
-  // landedPlanes = [];
-  // maximumCap;
-
-  // constructor(capacity = 10, maximumCap) {
-  //   this.capacity = capacity;
-  //   this.maximumCap = maximumCap;
-  // }
 
   constructor(capacity = 10) {
     this.landedPlanes = [];
     this.capacity = capacity;
-  }
+    this.Weather = new Weather();
+  };
 
   isFull = () => {
-    return this.capacity === this.landedPlanes.length;
+    if (this.capacity === this.landedPlanes.length)
+      return true;
```

Comment:
instead of returning booleans for all my functions, I should throw an error when the method isn't likely to be used in logical decisions.